src/App.jsx

```diff
@@ -1,19 +1,29 @@
 import { useEffect } from 'react';
-import { useDispatch } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 import { setToken, setUser } from '@/redux/authentication/authenticationSlice';
+import { getParachutes } from "@/redux/parachutes/parachutesSlice";
 import Router from "@/routing/Router"
 
 function App() {
   const dispatch = useDispatch();
+  const storedToken = localStorage.getItem('token');
+  const storedUser = localStorage.getItem('user');
+  const { token } = useSelector((store) => store.login)
+  const { parachutes } = useSelector((store) => store.parachutes)
 
   useEffect(() => {
-    const storedToken = localStorage.getItem('token');
-    const storedUser = localStorage.getItem('user');
+    console.log(storedToken);
     if (storedToken) {
       dispatch(setToken(storedToken));
       dispatch(setUser(storedUser));
     }
-  }, [dispatch]);
+  }, [dispatch, storedToken, storedUser]);
+
```

Comment:
Good efforts adding fetching parachutes at entry point. but when i refresh the page, it changes the loading state for parachute. this is happening because of dispatching `setToken` and `token` dependency.